client/src/components/atoms/Form/Input.module.scss

```diff
@@ -33,8 +31,8 @@
         width: 1.25rem;
         height: 1.25rem;
         top: 50%;
-        margin-top: -.6rem;
-        fill: rgba($brand-grey-light, .7);
+        margin-top: -0.6rem;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)